actors/miner/src/expiration_queue.rs

```diff
@@ -817,17 +822,15 @@ impl<'db, BS: Blockstore> ExpirationQueue<'db, BS> {
                     es,
                     epoch as ChainEpoch,
                 );
-                if group.sector_epoch_set.sectors.is_empty() {
-                    // by here it's guaranteed this is true:
-                    //      all_remaining.contains(&sector.sector_number) && es.on_time_sectors.get(sector.sector_number)
-                    // so group.sector_epoch_set.sectors should not be empty
-                    return Err(anyhow!("there's a bug in `group_expiration_set` function"));
+
+                if !group.sector_epoch_set.sectors.is_empty() {
+                    expiration_groups.push(group);
                 }
-                expiration_groups.push(group);
 
-                // no need to scan for this sector any more
-                Ok(false)
+                Ok(!all_remaining.is_empty())
```

Comment:
```suggestion
                Ok(epoch < new_end && !all_remaining.is_empty())
```

Reply:
I'm not sure _how_ important this will be.

Reply:
Done.